base/collect_database_test.go

```diff
@@ -10,13 +11,26 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var templatePath string = "../testdata/templates/ce.metrics.tpl"
-var tempMetricsPath string = "test-ce.metrics.yaml"
+const (
+	templatePath      = "../testdata/templates/ce.metrics.tpl"
```

Comment:
@sriumcp I wanted to change the `templatePath` so that it points to the template in `charts/iter8lib/templates` rather than relying on this old one in `testdata/templates` but I encountered some issues. (`testdata/templates` only contains this template by the way. I was always planning on using it temporarily.)

Firstly, because my metrics template uses the `include` action, I think I have to use the Helm's chart parser (chartutil?) as well as the `loader` package. I didn't know if you wanted these extra dependencies in `base/`. Secondly, this forces the test to rely on the `charts/` folder, and I don't know if `charts/` folder will always be in the same location, how to get that location, and if its contents will be complete. Thoughts?

Reply:
Use the `*.metrics.yaml` file for the unit test, and not the `*.metrics.tpl` file, since the unit tested functionality deals only with `yaml` and not `tpl`.